sdk/iOS/src/MSCoreDataStore.h

```diff
@@ -5,6 +5,7 @@
 #import <Foundation/Foundation.h>
 #import <WindowsAzureMobileServices/WindowsAzureMobileServices.h>
 #import <CoreData/CoreData.h>
+#import "MSSyncContext.h"
```

Comment:
I have a seperate PR fixing the #imports on this file so it doesn't use WindowsAzureMobileServices, but fine to leave this as is or just drop this change.